### DISCUSSION TITLE
Havinng trouble with WISDEM

### DISCUSSION THREAD

**Discussion by aravindvenkatachalapathy** [NatLabRockies/ROSCO#262] [General]
I have a problem when importing modules from ROSCO_toolbox. I running into this problem with WISDEM.

Traceback (most recent call last):
  File "D:\pythonvscode\venv\Lib\site-packages\wisdem\rotorse\rotor_elasticity.py", line 7, in <module>
    from wisdem.rotorse.precomp import PreComp, Profile, CompositeSection, Orthotropic2DMaterial
  File "D:\pythonvscode\venv\Lib\site-packages\wisdem\__init__.py", line 1, in <module>
    from wisdem.glue_code.runWISDEM import run_wisdem
  File "D:\pythonvscode\venv\Lib\site-packages\wisdem\glue_code\runWISDEM.py", line 11, in <module>
    from wisdem.glue_code.glue_code import WindPark
  File "D:\pythonvscode\venv\Lib\site-packages\wisdem\glue_code\glue_code.py", line 4, in <module>
    from wisdem.rotorse.rotor import RotorSE
  File "D:\pythonvscode\venv\Lib\site-packages\wisdem\rotorse\rotor.py", line 7, in <module>
    from wisdem.rotorse.rotor_elasticity import RotorElasticity
  File "D:\pythonvscode\venv\Lib\site-packages\wisdem\rotorse\rotor_elasticity.py", line 7, in <module>
    from wisdem.rotorse.precomp import PreComp, Profile, CompositeSection, Orthotropic2DMaterial
  File "D:\pythonvscode\venv\Lib\site-packages\wisdem\rotorse\precomp.py", line 21, in <module>
    from wisdem.rotorse._precomp import precomp as _precomp
ImportError: DLL load failed while importing _precomp: The specified module could not be found.

Process finished with exit code 1

What am i doing wrong here, i have installed WIDEM all the modules are there in the WISDEM env.

**Comment by dzalkind** [comment]
I'm not completely following your issue.  What example or script are you trying to run?  How has it been modified from the original? 

Is this an issue with WISDEM or the ROSCO_Toolbox?  What version of wisdem are you using?  `conda list | grep wisdem` will show you.

**Comment by aravindvenkatachalapathy** [comment]
I think ive solved the above issue by replacing this code "from wisdem.rotorse._precomp import precomp as _precomp" as "from wisdem.rotorse import precomp as _precomp" as Ive found that there is no Folder called _precomp inside WISDEM.

**Comment by aravindvenkatachalapathy** [comment]
<img width="988" alt="image" src="https://github.com/NREL/ROSCO/assets/43172847/f5714441-5b15-4a5d-9f94-d619bac90add">
Now Im running into this issue while Running example 5 for Openfast simulation

**Reply by dzalkind** [reply]
Looks like your call to openfast is not a valid command.  You need to set the executable/command as the `fastcall` here: https://github.com/NREL/ROSCO/blob/e3b7db779ad9e7bea5dea692e92f52b10116cf02/Examples/05_openfast_sim.py#L88

**Reply by aravindvenkatachalapathy** [reply]
<img width="1022" alt="image" src="https://github.com/NREL/ROSCO/assets/43172847/1f69fad5-8918-4851-9ade-09b892a65dba">
Same problem even after doing that.

**Reply by dzalkind** [reply]
You probably need the full path. If you type in the `fastcall` from where you run the script, does it run OpenFAST?
